api/src/entities/collection_mints.rs

```diff
@@ -82,6 +94,8 @@ pub enum Relation {
         on_delete = "Cascade"
     )]
     Collections,
+    #[sea_orm(has_many = "super::metadata_jsons::Entity")]
```

Comment:
This should be a has_one, yes?

Reply:
yeah , sea-orm-cli generates 1:M relation for every entity by default

Reply:
yeah , sea-orm-cli generates 1:M relation for every entity by default